### PR TITLE
Update Amz.py

### PR DESCRIPTION
**Pull request by nicballarini**
Added a function to automate the confirm address button and updated the main function to call said function and continue through the list of giveaways.

### CODE REVIEW

File: Amz.py [hunk 1]
```diff
@@ -222,8 +235,11 @@ def main():
             elif 'you won!' in title.text:
                 print(title.text)
                 print('You won!')
+                # Call the confirm_address func and 'continue' instead of 'breaking'
+                confirm_address(driver)
+                time.sleep(60)
                 you_won = True
-                break
+                continue
```
[the remark this answers is not in the document]
Comment (nicballarini) [reply Amz.py]: > Also, why do you sleep for 60 here? Seems like a long pause?
I did not have a chance to test this the automation, however, so this was my way of making sure to give ample time. You're right though, probably overkill.

>You are not reacting if the function call errors and returns False. This program was written to simply stop execution when giveaway is won, and I like your automation. However, maybe if it returns False, we should stop as normal? Something like this:

I created a new PR with this Amz.py update and also continuing to "break". Still allowing the person to see what they won without losing out by not confirming the address in time.

File: Amz.py [hunk 2]
```diff
@@ -103,6 +103,19 @@ def give_away_type(driver):
...
+    try:
+        driver.find_element(By.XPATH, '//input[@name="ShipMyPrize"]')
+        submit = WebDriverWait(driver, 20).until(
+			EC.element_to_be_clickable(
+				(By.XPATH, '//input[@name="ShipMyPrize"]'))
+		)
+        submit.click()
+    except:
+        return False
+
```
[the remark this answers is not in the document]
Comment (nicballarini) [reply Amz.py]: Can you help me understand this a bit? I used some of the previous funcs as a template:

ie
```
def youtube_video(driver):
    # Test if its an Youtube Video. If it is then execute
    try:
        js = 'document.evaluate("//div[@class=\'youtube-video\']/a", document.body, null, 9, null). singleNodeValue.click();'
        driver.execute_script(js)
        pause_small()
        submit = WebDriverWait(driver, 20).until(
            EC.element_to_be_clickable(
                (By.XPATH, '//button[@class="a-button a-button-primary youtube-continue-button"]'))
        )
        submit.click()
    except:
        return False
```

```
def instant_box(driver):
    # Test if its an Instant win box. If it is then execute
    try:
        driver.find_element(By.XPATH, '//*[@class="a-text-center box-click-area"]').click()
        pause_small()
    except:
        return False`
```

File: Amz.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (bitey-mouse) [reply Amz.py]: Oh, ok - so, looks like you should close this PR if you addressed this issue elsewhere?

**Comment by nicballarini** [comment]
Closing PR as I submitted another to merge 2 branches including these changes

New PR: https://github.com/Penlo/Amazon-Giveaway-Bot/pull/11

**Pull request closed without merging**